### ISSUE TITLE
reproducibility 

### ISSUE THREAD

**Issue by Bontempogianpaolo1** [bhrqw/AttriCLIP#2]
We integrated your code with cifar100 but we are not able to reproduce your same numbers... any suggestion?

**Comment by SanghyeokSon** [comment]
I can't reproduce both CIFAR100 and ImageNet100.
The task10 accuracy is lower by about 13.3 for ImageNet 100 and lower by about 3.4 for CIFAR100. 
I ran the published code as is, do I need to change the hyperparameter settings?

**Comment by Jay-IPL** [comment]
same here. Can not reproduce the numbers in the paper. On CIFAR100, I got a final acc=78.57 after training on 10 tasks. However, the paper reports 81.4. Such a huge gap.

I used --ckpt_path ViT-L/14 --train_batch 48

**Comment by srvCodes** [comment]
I can verify this too. Plus, interestingly, even zero-shot ContinualCLIP with ViT-L-14 reaches a final accuracy of 78.28% -- which means: (a) virtually no performance gain, (b) figures reported for ContinualCLIP in the paper do not seem to be that of ViT-L-14 but more of those produced from a ViT-B-16.

I have tried reaching out to the authors regarding this. Waiting for a response.

**Comment by bhrqw** [comment]
I have rewritten this project from its Mindspore version, but there may still be some issues with the code. I will thoroughly verify and update any necessary code. In the meantime, I will temporarily close this code to prevent any further impact from the current errors.
